microcosm_pubsub/chain/context.py

```diff
@@ -18,6 +18,9 @@ def __setitem__(self, key, value):
             raise ValueError(f"Key '{key}' already set")
         self.store[key] = value
 
+    def hard_set(self, key, value):
```

Comment:
In `microcosm` we use the name `assign` for this usage:  https://github.com/globality-corp/microcosm/blob/develop/microcosm/object_graph.py#L49